scripts/flush-dev-db.sh

```diff
@@ -1,18 +1,51 @@
 #!/bin/bash
```

Comment:
For me the name of this script is a little misleading: I could do without 'dev' in the name: suggest to me that it acts on the dev DB only.